vagrantconfig.yaml

```diff
@@ -9,4 +9,4 @@ memory: 256
 debug_mode: false
 
 # set this to "gui" if vagrant cannot connect
-boot_mode: none
+boot_mode: gui
```

Comment:
@willkg Having this line as null create connection Error with the VM. I could not run vagrant with having it None